sf-docker/special/sf-2-sf-synchronization/follower/docker-compose.yml

```diff
@@ -134,6 +159,7 @@ services:
       - S3Bucket__AccessKey
       - S3Bucket__SecretKey
       - NoSqlDataStorageDisabled
+      - SF_FACE_TEMPLATE_COMPATIBILITY_VERSION=1.40
```

Comment:
If this was intended, then Extraction__Algorithm should also be set to `accurate_mask` for extractor ?

Reply:
Good catch, this was indeed not intended.